rails/app/controllers/external_activities_controller.rb

```diff
@@ -186,7 +186,8 @@ def update
     end
 
     respond_to do |format|
-      if @external_activity.update_attributes(external_activity_strong_params(params[:external_activity]))
+      # allow for params[:external_activity] to not exist while other update_* params to update
+      if !params[:external_activity] || @external_activity.update_attributes(external_activity_strong_params(params[:external_activity]))
```

Comment:
This is to fix the tests.  In Rails 5 when you pass an empty hash like this (from the spec test):

`put :update, params: { :id => external_activity.id, :external_activity => {}, :update_projects => "true", :project_ids => [project1.id, project2.id] }`

params[:external_activity] is `nil` instead of `{}` in the controller.